openmetadata-ui/src/main/resources/ui/src/pages/TourPage/TourPage.component.tsx

```diff
@@ -31,13 +31,25 @@ const TourPage = () => {
     updateTourSearch,
   } = useTourProvider();
   const { t } = useTranslation();
+  const [isTourReady, setIsTourReady] = useState(false);
 
   const clearSearchTerm = () => {
     updateTourSearch('');
   };
 
   useEffect(() => {
     updateIsTourOpen(true);
+
+    const waitForElement = () => {
+      const el = document.querySelector('#feedWidgetData');
+      if (el) {
+        setIsTourReady(true);
+      } else {
+        setTimeout(waitForElement, 100);
+      }
+    };
+
+    waitForElement();
```

Comment:
there can we chance of running into infinite loop if element did not found? can we have better approach here?